src/click_select/index.es

```diff
@@ -10,7 +10,28 @@ class ClickSelect extends PureComponent {
     this.handleClick = this.handleClick.bind(this);
   }
 
-  handleClick(event) {
+  copyToClipboard() {
+    if (!document.queryCommandSupported) return false;
+
+    this.select();
+    let result;
+
+    try {
+      result = document.execCommand('copy');
+    } catch (err) {
+      result = false;
+    }
+
+    this.unselect();
+    return result;
```

Comment:
do you need to return result?

Reply:
yep, need to show in result if a copy to the clipboard was successed